src/ompl/geometric/planners/rrt/src/RRTstar.cpp

```diff
@@ -338,16 +394,15 @@ ompl::base::PlannerStatus ompl::geometric::RRTstar::solve(const base::PlannerTer
                 }
             }
 
-            if (prune_)
+            if (useNewStateRejection_)
             {
-                if (opt_->isCostBetterThan(costToGo(motion, false), bestCost_))
+                if (opt_->isCostBetterThan(solutionHeuristic(motion, false), bestCost_))
```

Comment:
In the literature, this isn't really the same thing as pruning. This is a rejection scheme that is performed on the output of the steer function. Made it a separate setting.